dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/insertmanager/insertionfilters/ValidRollingPeriodFilter.java

```diff
@@ -0,0 +1,31 @@
+package org.dpppt.backend.sdk.ws.insertmanager.insertionfilters;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.dpppt.backend.sdk.model.gaen.GaenKey;
+import org.dpppt.backend.sdk.semver.Version;
+import org.dpppt.backend.sdk.utils.UTCInstant;
+import org.dpppt.backend.sdk.ws.insertmanager.OSType;
+
+/**
+ * This filter checks for valid rolling period. The rolling period must always be in [1..144],
```

Comment:
What happens with keys that have rolling period `0`? Do they get corrected by `OldAndroid0RPModifier` before they come in here?

Reply:
Yes, as the modifier comes before the filter :)